src/app/shared/services/session/session.service.ts

```diff
@@ -0,0 +1,18 @@
+import { Injectable } from '@angular/core';
+import { constants } from '@constants';
+
+@Injectable({
+  providedIn: 'root'
+})
+
+export class SessionService {
+  constructor() { }
+
+  setAccessToken(accessToken: string): void {
+    localStorage.setItem(constants.USER_ACCESS_TOKEN_KEY, accessToken);
+  }
+
+  signedIn(): boolean {
+    return localStorage.getItem(constants.USER_ACCESS_TOKEN_KEY) != null;
```

Comment:
Alternative to do the same:

```suggestion
    return Boolean(localStorage.getItem(constants.USER_ACCESS_TOKEN_KEY));
```

Avoid doing as it's harder to read:

```suggestion
    return !!localStorage.getItem(constants.USER_ACCESS_TOKEN_KEY);
```